README.md

```diff
@@ -14,6 +14,11 @@ separate Charm.
 
 ## Usage
 
+The Prometheus Operator may be deployed using the Juju commandline as
+in
```

Comment:
The newline here looks strange, but could just be GitHub

Reply:
Just emacs markdown mode word wrapping the line because it 2 characters too long.